### PR TITLE
Export weekly non relevant contacts

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Generates contacts for participants who sent ["showtime_question", "greeting", "opt_in", "NC"] messages only in the past week so that we can resend a slightly tailored SMS ad with that week's question to gather relevant responses. 
Uses Core https://github.com/AfricasVoices/CoreDataModules/pull/227.

### CODE REVIEW

File: export_weekly_non_relevant_contacts.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+            for plan in PipelineConfiguration.RQA_CODING_PLANS:
+                if plan.dataset_name == target_dataset_name:
+                    for cc in plan.coding_configurations:
+                        codes = analysis_utils.get_codes_from_td(td, AnalysisConfiguration(plan.dataset_name,
+                                                                                           plan.raw_field,
+                                                                                           cc.coded_field,
+                                                                                           cc.code_scheme))
+                        for code in codes:
+                            if code.code_type == CodeTypes.NORMAL:
+                                has_relevant_message = True
```
Comment (as2388) [inline export_weekly_non_relevant_contacts.py:92]: CoreData has an implementation of relevance that might help here

File: export_weekly_non_relevant_contacts.py [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+            # Check for contacts who have only sent messages labeled as showtime_question", "greeting", "opt_in", "NC"
+            # in the target episodes. We will send a tailored follow up sms with the target episode question
+            for plan in PipelineConfiguration.RQA_CODING_PLANS:
+                if plan.dataset_name == target_dataset_name:
+                    for cc in plan.coding_configurations:
+                        codes = analysis_utils.get_codes_from_td(td, AnalysisConfiguration(plan.dataset_name,
+                                                                                           plan.raw_field,
+                                                                                           cc.coded_field,
+                                                                                           cc.code_scheme))
```
Comment (DanielMwendwa) [inline export_weekly_non_relevant_contacts.py:89]: `AnalysisConfiguration(plan.dataset_name, plan.raw_field, cc.coded_field, cc.code_scheme))`
I suggest you extract this into a variable and then pass to get_codes_from_td.

**Review by DanielMwendwa — APPROVED** [review]

File: export_weekly_non_relevant_contacts.py (hunk 1 above)
Comment (IsaackMwenda) [reply export_weekly_non_relevant_contacts.py]: Updated thanks.

File: export_weekly_non_relevant_contacts.py (hunk 2 above)
Comment (IsaackMwenda) [reply export_weekly_non_relevant_contacts.py]: Updated thanks

**Pull request merged**